pages/map.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Head from 'next/head';
+import styled from 'styled-components';
+
+class Map extends React.Component {
+  render() {
+    return (
+      <React.Fragment>
+        <Head>
+          <title>Texas Justice Initiative | Map</title>
+        </Head>
+        <MapIframe src="http://tji-map-viewer.s3-website.us-east-2.amazonaws.com/#4.77/31.67/-100.43" />
+      </React.Fragment>
+    );
+  }
+}
+
+export default Map;
+
+const MapIframe = styled.iframe`
+  height: calc(100vh - 153.375px);
```

Comment:
We don't explicitly set the height of the header on desktop, but Chrome calculated it at 153.375px. We should check this out in other browsers and consider defining it as a shared variable.